### DISCUSSION TITLE
disabling warnings

### DISCUSSION THREAD

**Discussion by castano** [shader-slang/slang#5802] [General]
Is there any way to disable specific warnings in Slang? I tried using #pragma warning:

```
    #pragma warning (push)
    #pragma warning (disable : 10002)
    ...
    #pragma warning (pop)
```

but pragma warning is not recognized.

This would be useful to disable warnings like this:

```
spark_bcn.h(1262): warning 10002: '0' prefix indicates octal literal
```

Octal literals are unusual, but there are legitimate uses for them. I can imagine other scenarios where this functionality would be useful. Has this been implemented already? Or in the absence of this, is there a way to disable specific warnings through the command line?

**Comment by saipraveenb25** [comment]
Hey! 

Yes, the Slang command line has options to disable warnings through `-Wno-<id>`. So adding `-Wno-10002` should silence this warning. 

Run `slangc.exe -help` to see more information.

**Reply by castano** [reply]
Thanks! It would be great to have support for pragma warning in the future, but for now that works.

**Reply by jkwak-work** [reply]
If the feature is needed, please file an issue so that we can properly schedule to implement it.
The implementation is not going to be trivial so we need to have an issue filed first.
